src/utils/tests/userHelpers.test.ts

```diff
@@ -0,0 +1,244 @@
+import { describe, expect, it } from 'vitest';
+import {
+  checkIfPersonHasNationalIdentificationNumber,
+  findFirstEmploymentThatMatchesAnActiveAffiliation,
+  getEmployments,
+} from '../user-helpers';
+import { CristinPerson, CristinPersonAffiliation, Employment } from '../../types/user.types';
+
+describe('getEmployments', () => {
+  const topOrgCristinId = '123.0.0.0';
+
+  const internalOrg = 'org/123.0.0.0'; // matches expected prefix
```

Comment:
FYI, dette med at samme prefiks betyr at enheten er på samme institusjon er en forutsetning vi bruker noen plasser selv om det ikke nødvendigvis er sant. I de aller fleste tilfeller stemmer det da

Der vi har vurdert det viktig nok gjør vi oppslag på hver enhet, og sjekker at den faktisk har samme `partOf`. Eneste plassen vi gjør det tror jeg er i `nviHelpers.ts` (se `hasChangedContributorsOrAffiliations()`).

Det kompliserer ganske mye da, også for testing, siden alle kall da må mockes og man må lage reelle organisasjoner.

Ikke noe endringsforslag her altså, bare for å forklare litt bakgrunn